helm/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. You can access immudb web console by running these commands:
+{{- if .Values.ingress.enabled }}
+  curl http{{ if $.Values.ingress.tls }}s{{ end }}://{{ .Values.ingress.hostname }}/
+  immuclient status -a {{ .Values.ingress.hostname }} -p 80 
+{{- else if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "immudb.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "immudb.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "immudb.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
+  echo http://$SERVICE_IP:{{ .Values.service.port }}
```

Comment:
`immuclient` example is missing from that and the container port example - is that intentional?

Reply:
I haven't (yet) tested immuclient with the loadbalancer and the container port configuration, I'll add it as soon as I have finished testing

Reply:
right, maybe we should then add some warning that this configuration was not fully tested then? (and that any feedback is welcome of course ;))

Reply:
fixed in commit 11b315756d7fd4748bbc1653d89e41ac1a9f5718